Rubberduck.Deployment/InnoSetup/Rubberduck.Installer.Build.iss

```diff
@@ -1066,7 +1109,16 @@ begin
     begin
       Log('Addin registration was declined because the user unchecked the checkbox');
     end;
+  end
+    else if CurPageID = WorkloadOptionPage.ID then
```

Comment:
I think this branch is missing an additional condition. You need something like this:
```
else if CurPageID = WorkloadOptionPage.ID then
  if WorkloadOptionPage.Values[0] then
  begin
    Log('Legacy workload initial config was requested and will be copied to the destination folder.');
  end
  else
  begin
    Log('Skipping legacy workload config because the option was left unchecked.');
  end;
end;
```